backend/src/resolvers/users.spec.js

```diff
@@ -71,7 +77,8 @@ describe('userMiddleware', () => {
 
     it('with too short name', async () => {
       const variables = {
-        id: 'u1'
+        id: 'u47',
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-10T17:59:52Z" title="Friday, May 10th 2019, 7:59:52 pm +02:00">May 10, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -71,7 +77,8 @@ describe('userMiddleware', () => {
 
     it('with too short name', async () => {
       const variables = {
-        id: 'u1'
+        id: 'u47',
```


I think I have to give myself a "Request Changes" review here. I told @ulfgebhardt not to create test cases (==specification) for behaviour that we actually don't want to support, e.g. passing `id` from the client for `CreateUser` mutation. I'll update that as soon as possible. I think, at first, I thought I have to pass the `id` to make it happen. That's why it's there.